src/modules/filters/globalFilters.ts

```diff
@@ -11,35 +11,44 @@ import {
 import { IProduct } from '../interfaces/product.interface';
 import { DualSlider } from './sliders';
 import { Search } from './search';
+import { Sort } from './sort';
 
 //I'll change it later when all layout is refactored
 const wrapperFiltres: Element = document.querySelector('.filters')!;
 const wrapperSliders: Element = document.querySelector('.sliders')!;
+const WRAPPER_SORT: Element = document.querySelector('.sort__wrapper')!;
```

Comment:
Why is this capitalized suddenly?

Reply:
Good question)
I used upper-case in app.ts and then decided to use it here as well. Do you think it makes sense to use upper-case for constants?

Reply:
It's up to you. Better check with style guides. But anyway: either all capitalized or all camelCase.